src/nvim/profile.c

```diff
@@ -943,3 +943,43 @@ void time_msg(const char *mesg, const proftime_T *start)
   g_prev_time = now;
   fprintf(time_fd, ": %s\n", mesg);
 }
+
+/// Initializes the time time_fd stream used to write startup times
+///
+/// @param startup_time_file the startuptime report file path
+/// @param process_name the name of the current process to write in the report. Example: "Main" or "TUI"
+void time_init(const char *startup_time_file, const char *process_name)
```

Comment:
Better name these `startuptime_init()` and `startuptime_finish()`

Reply:
Update: I didn't realized there are already many other functions with `time_` prefix. Maybe `time_` prefix is fine then. (although not straightforward enough)

Reply:
yeah, I agree. Just reverted.